src/apps/testapps/testCompactCells.c

```diff
@@ -70,6 +70,24 @@ SUITE(compactCells) {
         free(sunnyvaleExpanded);
     }
 
+    TEST(res0children) {
+        H3Index parent;
+        setH3Index(&parent, 0, 0, 0);
+
+        int64_t arrSize;
+        t_assertSuccess(H3_EXPORT(cellToChildrenSize)(parent, 1, &arrSize));
+
+        H3Index *children = calloc(arrSize, sizeof(H3Index));
```

Comment:
This one also needs a `free` (which is why Valgrind is complaining)

Reply:
just saw it ;)